ci/cloudbuild/notifiers/bigquery/deploy.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# There are a few commmands that need to be run in order to deploy a change to
+# the bigquery.yaml file, and this script runs those commands.
+
+set -euo pipefail
+
+source "$(dirname "$0")/../../../lib/init.sh"
+source module ci/lib/io.sh
+cd "${PROGRAM_DIR}"
+
+readonly BUCKET="gs://cloud-cpp-testing-resources-configs/google-cloud-cpp"
```

Comment:
nit: Presumably this bucket was created by you?  I am slightly worried that our build setup require so many manual steps, many of which are probably undocumented and forgotten. Not specific to this PR, just something for us to discuss elsewhere.

Reply:
Yes, and agreed. As I mentioned above, setting up this notifier required about 10 manual steps once. It was a big PITA. Not sure how to make this better.